core/modules/user/user.module

```diff
@@ -1170,6 +1198,13 @@ function user_login_block($form) {
     '#required' => TRUE,
     '#weight' => 2,
     '#password_toggle' => TRUE,
+    '#attributes' => array(
+      // Add attributes to field to prevent spell-jacking and unwanted automatic
+      // changes caused by the browser.
+      'autocapitalize' => 'none',
+      'autocorrect' => 'off',
+      'spellcheck' => 'false',
+    ),
```

Comment:
This block simply adds the desired attributes to both name and password for this specific high-level form.